apps/system/linux/machine/generic/platform_info.c

```diff
@@ -26,6 +26,7 @@
 #include <metal/utilities.h>
 #include <openamp/remoteproc.h>
 #include <openamp/rpmsg_virtio.h>
+#include <errno.h>
```

Comment:
This one is also needed? seems related to Linux not FreeRTOS

Reply:
I decided to add it to this file so that everything that uses an errno value directly includes the relevant header. This makes it clear that the use of the errno constants is intended.

Reply:
Right!  i crosschecked the code, i did not seen any instance that you not address